vllm/config/compilation.py

```diff
@@ -649,15 +653,8 @@ def set_splitting_ops_for_v1(self):
 
     def set_splitting_ops_for_inductor_graph_partition(self):
         assert self.use_inductor_graph_partition
-        use_inductor_graph_partition_msg = (
-            "When use_inductor_graph_partition=True, splitting_ops "
-            "are ignored and set to an empty list. Instead, "
-            '"tags=(torch._C.Tag.cudagraph_unsafe, )," is '
-            "used to annotate custom ops for graph partition."
-        )
-        if self.splitting_ops is not None and len(self.splitting_ops) > 0:
-            logger.warning_once(use_inductor_graph_partition_msg)
-        self.splitting_ops = []
+        if not self.splitting_ops:
+            self.splitting_ops = list(self._attention_ops)
```

Comment:
why set non-empty splitting_ops for inductor graph partition?

Reply:
Oh I see. splitting_ops are used for both fx partition and inductor partition now.

Reply:
Yeah this is the whole point of this PR, respect `splitting_ops` in both cases